Libraries/src/Amazon.Lambda.RuntimeSupport/Helpers/Logging/JsonLogMessageFormatter.cs

```diff
@@ -0,0 +1,301 @@
+﻿#if NET6_0_OR_GREATER
+using System;
+using System.Buffers;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Amazon.Lambda.RuntimeSupport.Helpers.Logging
+{
+    /// <summary>
+    /// Formats the log message as a structured JSON log message.
+    /// </summary>
+    public class JsonLogMessageFormatter : AbstractLogMessageFormatter
+    {
+        private static readonly UTF8Encoding UTF8NoBomNoThrow = new UTF8Encoding(encoderShouldEmitUTF8Identifier: false, throwOnInvalidBytes: false);
+
+        // Options used when serializing any message property values as a JSON to be added to the structured log message.
+        private JsonSerializerOptions _jsonSerialiazationOptions;
```

Comment:
typo in `_jsonSerialiazationOptions`. should be `_jsonSerializationOptions`

Reply:
Fixed